test-clients/src/main/resource/contract/contracts/RandomGeneratePrecompile/RandomGeneratePrecompile.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import "./IRandomGenerate.sol";
+
+contract RandomGeneratePrecompile {
```

Comment:
This solidity file and the `IRandomGenerate.sol` should have their source of truth in the [smart-contracts repo](https://github.com/hashgraph/hedera-smart-contracts).
Can you open a ticket there with an eventual PR to add these 2 solidity files to a `math-precompile` folder

Reply:
yes, did not know that. Will open ticket and a PR.

Reply:
Created this [issue](https://github.com/hashgraph/hedera-smart-contracts/issues/31), but I am not able to assign to me. Will open a PR once this PR is finalized.